### DISCUSSION TITLE
User Roles Permission: Can't see other users + Migrating from another Time Tracking Tool

### DISCUSSION THREAD

**Discussion by g-rakovszky** [kimai/kimai#2884] [Questions & Answers] [answered]
Good day,

Our company is currently testing Kimai.
I created some test accounts. But unfortunatelity the roles "teamlead" or any other role except the System-admin can't see  any other users. Even though I activated the "view_user" option in.
My goal is, that the teamlead is able to edit the hourly rate of an user.
Am I missing something?

Another thing.
According to the documentation, it's sufficent if we export all our data to csv files and import it directly in kimai. 
Is it really that easy or am I overlooking something important here?

Cheers

**Comment by kevinpapst** [comment, marked as answer]
Roles are bound to functional permissions. These permissions limit what features you can use.

Teams allow to open up visibility of content. If you create a team the assigned teamleads will see the other users in the team. The teamleads do not have to have the role "teamlead" but in most cases should. Becuase the role teamed allows to see the timesheets of other users.

There is an exception, the permission `view_all_data`- but this one is rather dangerous and should not be used in most environments.

You could create one team, add all users and set all as teamlead. Then everyone will see everyone else.

> According to the documentation, it's sufficent if we export all our data to csv files and import it directly in kimai.
> Is it really that easy or am I overlooking something important here?

Whether this is easy or not depends on the data you have. You have to create at least a matching CSV structure. 
There are many companies who migrated successful to Kimai with this approach. Other companies used the API or even direct database access. So, not sure what you want to know exactly.

**Reply by g-rakovszky** [reply]
Thanks for your response.
You helped me a lot.
